source/tutorials/write-delete-documents.txt

```diff
@@ -0,0 +1,186 @@
+.. _java-rs-write-delete-docs:
+
+================
+Delete Documents
+================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: code examples, write, remove
+
+Overview
+--------
+
+In this guide, you can learn how to use the {+driver-short+} to delete
+documents from a MongoDB collection by performing delete operations.
+
+A delete operation deletes one or more documents from a MongoDB collection.
+You can perform a delete operation by using the ``deleteOne()`` or
+``deleteMany()`` methods.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``sample_restaurants.restaurants`` collection
+from the :atlas:`Atlas sample datasets </sample-data>`. 
+
+.. TODO:
+.. Once get started ticket merged in, link the get started tutorial below
+
+.. To learn how to create a
+.. free MongoDB Atlas cluster and load the sample datasets, see the
+.. :ref:`<java-rs-getting-started>` tutorial.
+
+.. TODO: When write insert methods merged in, add subscriber methods section includes within subscriber methods section
+
+.. Subscriber Methods
+.. ~~~~~~~~~~~~~~~~~~~~
+
+Delete Operations
+-----------------
+
+You can perform delete operations in MongoDB by using the following methods:
+
+- ``deleteOne()``, which deletes *the first document* that matches the search criteria
+- ``deleteMany()``, which deletes *all documents* that match the search criteria
+
+Each delete method requires a **query filter**, which specifies the
+search criteria that determine which documents to select for removal.
+For more information about query filters, see the 
+:manual:`Query Filter Documents section </core/document/#query-filter-documents>` in
+the MongoDB Server manual.
+
+Delete One Document
+-------------------
+
+To delete a single document to a MongoDB collection, call the ``DeleteOne()``
+method and pass in your query filter. After you call the
+``insertOne()`` method, chain the ``subscribe()`` method and pass in a subscriber.
```

Comment:
Would a brief explanation of subscribers be helpful here? Not sure if this concept is introduced anywhere else in these docs. If subscribers are a common concept in async Java we expect readers to know then maybe not.

Reply:
I have a todo above where I will (when the write insert methods page is merged in) add a includes about subscribers and then link to a more in depth page about the concept of subscribers:

`TODO: When write insert methods merged in, add subscriber methods section includes within subscriber methods section

.. Subscriber Methods
.. ~~~~~~~~~~~~~~~~~~~`

This would be above delete operations!